blog/2022-12-01-governance-changes.md

```diff
@@ -6,8 +6,7 @@ title: "Governance changes"
 
 Sandra Fauconnier has been OpenRefine's project director since February 2022, and has been working on many fronts in this capacity: leading a reflection on the transformation of the project's governance model, running the biyearly user survey, coordinating our participation in the Outreachy internship programme, migrating from Google Groups to our new Discourse forum, and many more project management tasks behind the scenes. Given her close ties with the Wikimedia and Wikibase communities, she also strengthened the project's ties with those partners.
 
-Sandra has decided to step down from the project director role to be able to fully focus on her work on the Wikimedia movement. She plans to still be involved with OpenRefine, for instance to run training workshops about the
-Wikimedia Commons integration project she lead over the past year. As of December 1st, she has transfered back the general project governance responsibilities to the advisory committee, consisting of Martin Magdinier, Jan Ainali and Antonin Delpeuch.
+Sandra has decided to step down from the project director role, returning to her previous work in Linked Open Data in the cultural sector and the Wikimedia movement. She plans to still be involved with OpenRefine's community and will continue to advocate for the tool and to give trainings. As of December 1st, she has transfered back the general project governance responsibilities to the advisory committee, consisting of Martin Magdinier, Jan Ainali and Antonin Delpeuch.
```

Comment:
Please change to:

Sandra has decided to step down from the project director role, returning to her previous work in Linked Open Data in the cultural sector and the Wikimedia movement. As of December 1st, she has transfered back the general project governance responsibilities to the advisory committee, consisting of Martin Magdinier, Jan Ainali and Antonin Delpeuch.

(Can't promise that sentence either. Looking at the actions and decisions made by the project contributors, and their key interests: the communities I serve will need to watch carefully if OpenRefine will still serve their needs in the future, and whether the project is worth their investment.)